ParallelMatrixMultiplication/ParallelMatrixMultiplication/MatrixUserInterface.cs

```diff
@@ -0,0 +1,121 @@
+﻿// <copyright file="MatrixUserInterface.cs" company="Kalinin Andrew">
+// Copyright (c) Kalinin Andrew. All rights reserved.
+// </copyright>
+
+namespace ParallelMatrixMultiplication
+{
```

Comment:
```suggestion
namespace ParallelMatrixMultiplication;
```

Используйте file-scoped неймспейсы, чтобы не создавать лишних уровней вложенности